stl/src/taskscheduler.cpp

```diff
@@ -52,7 +52,7 @@ namespace Concurrency {
 #else // ^^^ defined(_CRT_APP) ^^^ // vvv !defined(_CRT_APP) vvv
                 HMODULE _Result;
                 if (::GetModuleHandleExW(_Flags, _Addr, &_Result) == 0) {
-                    return 0;
+                    return nullptr;

```

Comment:
The `#if defined(_CRT_APP)` case above should also be changed.

Reply:
done